### PR TITLE
Enhancement for the bootstrap of admission

### PR DESCRIPTION
**Pull request by chendave**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind bug
/kind cleanup

**What this PR does / why we need it**:
enhancement for the bootstrap of admission

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by chendave** [comment]
cc @fisherxu

File: build/admission/gen-admission-secret.sh [hunk 1]
```diff
@@ -32,24 +32,52 @@ DNS.2 = ${SERVICE}.${NAMESPACE}
...
+      # create the secret with CA cert and server cert/key
+      kubectl create secret generic ${SECRET} \
+          --from-file=tls.key=${CERTDIR}/server.key \
+          --from-file=tls.crt=${CERTDIR}/server.crt \
+          --from-file=ca.crt=${CERTDIR}/ca.crt \
+          -n ${NAMESPACE}
+  fi
+
+  # create other objects needed for admission
+  for resource in $(ls *.yaml); do kubectl create -f $resource; done
```
Comment (fisherxu) [inline build/admission/gen-admission-secret.sh]: i'd prefer split the deploy and cert generate in different scripts, and in the long run, we should centralize the deployment, using helm or keadm.

File: build/admission/README.md [hunk 2]
```diff
@@ -23,12 +23,4 @@ cd build/admission
...
 
-Second, we create k8s resources from the manifests in name order. Before
-creating, check the content of each manifest to make sure it meets your
-environment.
-
-```bash
-for resource in $(ls *.yaml); do kubectl create -f $resource; done
-```
-
-Last, please use `kubectl get pods -nkubeedge` to check whether the admission run successfully.
```
[the remark this answers is not in the document]
Comment (chendave) [reply build/admission/README.md]: It's been done here:
https://github.com/kubeedge/kubeedge/pull/1352/files#diff-16d3ec35bbf9a00da3ece67541aecdceR49

File: build/admission/gen-admission-secret.sh (hunk 1 above)
Comment (chendave) [reply build/admission/gen-admission-secret.sh]: reverted the change here.

File: build/admission/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chendave) [reply build/admission/README.md]: ignore my above comment, I have reverted this change as well. Good catch!

**Pull request merged**